dvc/utils/pkg.py

```diff
@@ -1,63 +1,17 @@
 from dvc.utils import is_binary
 
 
-def is_conda():
+def check_build_patch():
     try:
         from .build import PKG  # patched during conda package build
 
-        return PKG == "conda"
+        return PKG
     except ImportError:
-        return False
+        return None
 
 
-def get_linux():
-    import distro
-
+def get_package_manager():
     if not is_binary():
```

Comment:
homebrew formula is also not binary, so this won't work

Reply:
We actually no longer need this check, as all our bin packages include `build.py`.